modin/core/storage_formats/pandas/utils.py

```diff
@@ -38,7 +39,12 @@ def get_default_chunksize(length, num_splits):
     )
 
 
-def compute_chunksize(df, num_splits, default_block_size=32, axis=None):
+def compute_chunksize(
+    df,
+    num_splits,
+    default_block_size=MinNumElementsToStartNewPandasPartition.get(),
```

Comment:
```suggestion
    default_block_size=None,
```
and down below in the function body do smth like
```python
default_block_size = MinNumElementsToStartNewPandasPartition.get() if default_block_size else default_block_size
```
that way it would allow the user to tune this variable during their workloads and to have a dynamic split.

Reply:
Done, except I swapped the `if` and `else` in your suggestion for the function body. I think you meant that we should fall back to the environment variable if `default_block_size` isn't specified.